keyboards/mariorion_v25/readme.md

```diff
@@ -0,0 +1,30 @@
+# MariOrion v2.5
+
+![mariorion_v25_1](https://i.imgur.com/3OrUK2ph.jpeg)
+![mariorion_v25_2](https://i.imgur.com/OvMBHOlh.jpeg)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Maria](https://github.com/toril940)
+* Hardware Supported: MariOrion v2.5 PCB
+* Hardware Availability: *Links to where you can find this hardware*
```

Comment:
Is there a link available to purchase this PCB or are there related kicad/gerber files?

Reply:
the PCB purchase link I need to wait for hiney to stick up since I'm going through him HAHA I'll edit it in once he's made a listing~